File: javascript/lesson 7/graph.js
class Graph {
    constructor(capacity) {
        this.adj = new Array(capacity);
        for (let i = 0; i < capacity; i++) {
            this.adj[i] = [];
        }
    }

    get(v) {
        return this.adj[v];
    }

    add(v, w) {
        this.adj[v].push(w);
        this.adj[w].push(v);
    }

    size() {
        return this.adj.length;
    }
}

File: python/lesson 7/graph.py
class Graph:
    def __init__(self, capacity: int):
        self.adj = [[]] * (capacity)
        for i in range(0, capacity-1):
            self.adj[i] = []
        
    def size(self) -> int:
        return len(self.adj)
    
    def get(self, v: int) -> List[int]:
        return self.adj[v]
    
    def add(self, v: int, w: int):
        self.adj[v].append(w)
        self.adj[w].append(v)
